scripts/compile/compile-stage.ps1

```diff
@@ -59,6 +61,11 @@ $Projects | ForEach-Object {
     }
 }
 
+if (Test-Path $binariesOutputDir) {
+    cp $binariesOutputDir\* $OutputDir\bin -force -recurse
+    Remove-Item $binariesOutputDir -recurse
```

Comment:
I like that solution! 
It would separate the files produced by compile from the files produced by publish. Nice and clean :)